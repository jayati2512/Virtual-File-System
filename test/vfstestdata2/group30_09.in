createvfs vfs 4
mountvfs vfs
makedir / vehicle
makedir / transport
makedir /vehicle 4wheeler
makedir /vehicle 2wheeler 
makedir /vehicle/2wheeler hero 
makedir /transport road
makedir /transport rail
makedir /transport air
makedir /transport/rail platform
makedir /vehicle/4wheeler benz
listdir / 1 starting
deletedir /transport
addfile /vehicle vehicleInfo /usr/include/wait.h
listdir / 1 afterdel
