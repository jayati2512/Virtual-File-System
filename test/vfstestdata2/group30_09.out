createvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
listdir_SUCCESS
deletedir_FAILED DIRECTORY_IS_NOT_EMPTY
addfile_SUCCESS
listdir_SUCCESS
